dev-utils/rollup-plugin-copy-watch/package.json

```diff
@@ -2,6 +2,7 @@
   "name": "@polypoly-eu/rollup-plugin-copy-watch",
   "main": "index.js",
   "private": true,
+  "scripts":{ "test": "jest"},
```

Comment:
Nit: This drives me a bit crazy, I'd go for either:

`"scripts": { "test": "jest" },`

Or:

```
"scripts": {
    "test": "jest"
},
```